_docs/common-issues.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: docs
+title: Common Issues
+permalink: /docs/common-issues/
+---
+
+These issues are often reported at the support forum or discord. Cannot find your problem? Go to Troubleshooting for general help.
+
+## Package http-interop/http-middleware is abandoned
+
+This is a warning and you can safely ignore it. The dependency will be fixed in a future version.
+
+## Internal Server Error / Permission denied
+
+Flarum needs permission to write files (cache, avatars). You may get error messages if your folders are not writeable by the PHP user. 
+
+
+**Solution A**
+
+First, find out which user PHP is running under (usually `www-data`):
+
+```raw
+ps uxa | grep php-fpm
+```
+
+Then, as `root`, change the ownership to this user:
+
+```raw
+chown -R www-data:www-data assets storage/*
+```
+
+
+**Solution B**
+
+If you use shared hosting, an alternative solution is to change the file permissions:
+
+```raw
+chmod 777 assets assets/* storage/*
+```
+
+## "Something went wrong while trying to load the full version of this site"
+
+A bug in Flarum causes the asset files to not regenerate properly. It happens when you enable specific extensions. This will be fixed in Flarum in a future version. For now, you solve it by running in a shell:
+
+```raw
+php flarum cache:clear
+rm assets/rev-manifest.json
+```
+
+And force-reload the page in your browser (e.g. `ctrl-shift-R`). NB. this is only required when you enable extensions. Your forum visitors will not notice it. 
+
+## Icons are squares
+
+This happens when your configured URL differs from the one you are using right now. Check whether the URL in `config.php` is the same. Also check for a possible `http` <> `https` mismatch.
+
+## Slow
+
+There could be various reasons why your Flarum feels slow. A normal page load should happen in under 1 second (check in your browsers developer tools).
+
```

Comment:
I would also add here that it could be debug mode is enabled in config.php, which means assets don't get minified (so they are in the MB size range and could take a while to download). Simply disable debug mode and clear the cache to regenerate minified assets.